### ISSUE TITLE
Create istio 1.6.0-alpha.0

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#22230]
We should be releasing alpha builds every 2ish weeks before code freeze

**Comment by howardjohn** [comment]
Please let me know if you need and help getting this started - I think the 1.5 team did a handoff meeting, but happy to help as well

**Comment by howardjohn** [comment]
This is blocking testing of some changes I have made to the release pipeline is why I care

**Comment by jacob-delgado** [comment]
@howardjohn We did have a meeting where we discussed the hand-off (https://docs.google.com/document/d/1ejb4HvPYlEchkD5klDKTMIzFZrHGJ5n5lNij8NjRE-Y/edit).

Given that this is blocking you and 1.6 is on a short timeline, let's setup some time tomorrow where you, @brian-avery and I can walk through the process. This part is still not clear to me. I'm available from all day tomorrow (I start work around 9:30 MT and don't finish until 6pm), except during the TOC.
